src/libraries/System.Net.Http/ref/System.Net.Http.cs

```diff
@@ -213,12 +213,22 @@ public partial class HttpRequestMessage : System.IDisposable
         public System.Net.Http.Headers.HttpRequestHeaders Headers { get { throw null; } }
         public System.Net.Http.HttpMethod Method { get { throw null; } set { } }
         public System.Collections.Generic.IDictionary<string, object?> Properties { get { throw null; } }
+        public HttpRequestOptions Options { get { throw null; } }
         public System.Uri? RequestUri { get { throw null; } set { } }
         public System.Version Version { get { throw null; } set { } }
         public void Dispose() { }
         protected virtual void Dispose(bool disposing) { }
         public override string ToString() { throw null; }
     }
+
+    public readonly struct HttpRequestOptionsKey<TKey>
+    {
+        public HttpRequestOptionsKey(TKey key) {}
+        public TKey Key { get { throw null; } }
+    }
```

Comment:
This isn't what was approved.  The issue shows this:
```C#
    public readonly struct HttpRequestOptionsKey<TValue>
    {
        public HttpRequestOptionsKey(string key);
        public string Key { get; }
    }
```

Reply:
Addressed in recent commits